### ISSUE TITLE
Can not connect to DB 

### ISSUE THREAD

**Issue by khanakhun** [hgourvest/node-firebird#257]
I need Help Connecting to a firebird DB, I'm new to firebird so I'm facing issues, any help will be great 

var options = {}

options.host = 'localhost'
options.port = 3050
options.database = 'devdb'
options.user = 'SYSDBA'
options.password = 'root'
options.lowercase_keys = false 
options.role = null 
options.pageSize = 4096

Firebird.attach(options, function (err, db) {
	if (err) throw err

	// db = DATABASE
	db.query('SELECT * FROM TABLE', function (err, result) {
		// IMPORTANT: close the connection
		console.log(result)
		db.detach()
	})
})

**Comment by Tomas2D** [comment]
Hey, without posting an error message, nobody can helps you :-)
